.kokoro/presubmit/downstream-compatibility-spring.sh

```diff
@@ -47,10 +47,16 @@ pushd spring-cloud-gcp/spring-cloud-generator
 
 
 # Generate showcase autoconfig
+pushd spring-cloud-generator
+# The script is not executable for non-owners. Here we manually chmod it.
+# TODO(diegomarquezp): remove this line after
+# https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/3183 is merged
+chmod 755 ./scripts/generate-showcase.sh
```

Comment:
@blakeli0 @JoeWang1127  I fixed the spring check in this PR as well since I thought it's small enough. There is a TODO to be solved next week, once https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/3183 is merged